src/tests/test_dynamic_api/test_filters.py

```diff
@@ -74,6 +74,28 @@ def test_serializer_has_nested_table(
         result = filterset.filter_queryset(Parkeervakken.objects.all())
         assert result.count() == 1, result
 
+    @staticmethod
+    def test_numerical_filters(parkeervakken_parkeervak_model, filled_router):
+        """Test comparisons on numerical fields."""
+
+        p1 = parkeervakken_parkeervak_model.objects.create(id="p1", aantal=1)
+        p2 = parkeervakken_parkeervak_model.objects.create(id="p2", aantal=2)
+        p5 = parkeervakken_parkeervak_model.objects.create(id="p5", aantal=5)
+
+        for op, filter_aantal, expect in [
+            ("lte", 1, [p1]),
+            ("gt", 3, [p5]),
+            ("lt", 3, [p1, p2]),
+            ("gte", 6, []),
+        ]:
+            response = APIClient().get(
```

Comment:
zal wel vaker zo gedaan zijn, maar hier hebben we ook een `api_client` fixture voor.